WebRoot/js/zimbraMail/share/view/dialog/ZmRevokeShareDialog.js

```diff
@@ -72,8 +72,8 @@ function(share) {
 	var isGuestShare = share.isGuest();
 	var isAllShare = share.grantee && (share.grantee.type == ZmShare.TYPE_ALL);
 
-	var params = isPubShare ? ZmMsg.shareWithPublic : isGuestShare ? share.grantee.id : isAllShare ? ZmMsg.shareWithAll :
-					(share.grantee.name || ZmMsg.userUnknown);
+	var params = isPubShare ? ZmMsg.shareWithPublic : isGuestShare ? AjxStringUtil.htmlEncode(share.grantee.id) : isAllShare ? ZmMsg.shareWithAll :
+					(AjxStringUtil.htmlEncode(share.grantee.name) || ZmMsg.userUnknown);
 	this._confirmMsgEl.innerHTML = this._formatter.format(params);
```

Comment:
For me, the fix was good to be at line 77, just because this is the moment we are manipulating a DOM where XSS attacks actually happen.